lib/aliquot-pay.rb

```diff
@@ -41,7 +40,7 @@ def extract_root_signing_keys
     {
       'keys' => [
         'protocolVersion' => @protocol_version,
-        'keyValue'        => key,
+        'keyValue'        => key
       ]
```

Comment:
This looks like a hash rather than an array? :thinking:

I know, not invented in this PR.

Reply:
https://rubystyle.guide/#no-trailing-array-commas

Reply:
I didn't question the comma. The removal of it is great :smile: 

It's `[ 'a' => :b, 'c' => :D ]` vs `{ 'a' => :b, 'c' => :D }`, but it's not important. Apparently it works.

Reply:
```ruby
irb(main):001:0> [ 'a' => :b, 'c' => :d ]
=> [{"a"=>:b, "c"=>:d}]
irb(main):002:0> { 'a' => :b, 'c' => :d }
=> {"a"=>:b, "c"=>:d}
```

:bulb: